user/views.py

```diff
@@ -44,28 +45,47 @@ def privacy_delete_trip(date):
     return redirect('privacy'), 204
 
 
-@app.route('/profile/', methods=['patch'])
+@app.route('/profile/')
```

Comment:
Why do you remove the `methods=['patch']`? We can't post with `patch` anymore then.
How would post the changes?

Reply:
as you made this comment on your own code I'm not sure about what I have to do with this 😛